content/docs/pipelines/deployment-alternatives.md

```diff
@@ -0,0 +1,139 @@
+
+
+## Deploying Kubeflow Pipelines standalone to an existing cluster
+
+1. Deploy latest version of Kubeflow Pipelines:
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
+kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+```
+
+2. Get the URL for the Kubeflow Pipelines UI :
+```
+kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com 
+```
+
+## Deploying Kubeflow Pipelines standalone from scratch
+
+1. Prepare a Kubernetes cluster:
+
+    You have many options from cloud service providers. For Google Cloud
+    Platform (GCP), you can reference [Creating a cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster).
+
+    You can also set up your own Kubernetes cluster.
+
+    Cluster minimum requirement: node pool with at least 2 nodes and 2 CPUs per node.
+
+1. Download kubectl CLI tool:
+
+    You can get kubectl from [its official doc](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+1. Configure kubectl to talk to your cluster:
+
+    For GCP, you can reference [Configuring cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl).
+
+1. Deploy latest version of Kubeflow Pipelines standalone to your cluster:
+
+    ```
+    export PIPELINE_VERSION={{% kfp-latest-version %}}
+    kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    ```
+
+    Kubeflow Pipelines applications take a while (~3 minutes) to start.
+
+1. Get public URL of Pipelines UI and use it to access Kubeflow Pipelines:
+    ```
+    kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com 
+    ```
+
+
+## Customization
+Customization can be done through kustomize [overlays](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#overlay). 
+
+Note - The instruction below assume you installed kubectl v1.14.0 or later, which has native support of kustomize.
+To get latest kubectl, visit [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+For the following instructions, first clone [Kubeflow Pipelines repo](https://github.com/kubeflow/pipelines),
+and use it as working directory.
+
+### Deploy on GCP with CloudSQL and Google Cloud Storage
+This is recommended for production environments. See
+[here](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/env/gcp) for more details. 
+
+### Change deployment namespace
+To deploy Kubeflow Pipelines standalone in namespace FOO:
+
+- Edit [dev/kustomization.yaml](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/env/dev/kustomization.yaml)
+or [gcp/kustomization.yaml](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/env/gcp/kustomization.yaml)
+namespace section to FOO.
+- Then run 
+```
+kubectl apply -k manifests/kustomize/env/dev # or other env of your choice
```

Comment:
this is confusing to read. maybe just paste the other command. kubectl apply -k manifests/kustomize/env/gcp

Reply:
thanks, will change